src/components/Search/SearchComponent.tsx

```diff
@@ -101,12 +94,13 @@ function SearchComponent({
           </Tooltip>
         </InputLabel>
         {/**** Edit Button ****/}
-        {hasEditButton && !formIsDisplayed && (
+
+        {/* temporary until global edit button PR */}
+        {hasEditButton && !formIsDisplayed && isBaseComp && (
```

Comment:
It would be better to have another boolean controlling this.
But the best name for this boolean would be `hasEditButton`, right? 😅 and it clashes with the other one, and frankly I don't want to nitpick about that anymore. So let's keep it this way now.

Reply:
I'll simply remove it from the search component. That's why I said it was temporary until the edit button PR.